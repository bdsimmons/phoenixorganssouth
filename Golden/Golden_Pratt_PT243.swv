<?xml version="1.0" encoding="UTF-8"?>
<SWiSHvideo version="2.0" fileVersion="1.0" build="2005.12.04"
 originalFolder="C:\Documents and Settings\David\My Documents\My Videos\FlashMovies\Golden"
>
	<video
		file="..\..\Golden_Pratt_PT243.wmv"
		bitrate="192000"
		quality="80"
		keyInterval="10"
		deinterlace="0"
		deepmotiondetection="0"
		outputFps="20.0"
		xResize="100%"
		yResize="100%"
		crop="0,0,0,0"
	/>
	<audio
		alt=""
		outputFlag="1"
		fadeOutTime="0"
		frequency="22050"
		mono="0"
		compression="1"
		param="64 (LAME MP3)"
		saveTo=""
	/>
	<output
		file=".\Golden_Pratt_PT243.swf"
		format="0"
		borderWidth="0"
		bgColor="0"
		stop="3"
		url=""
		target=""
		html="1"
		events="0"
		controls="Ultra Minimalist"
	/>
	<variables>
		<var name="umTrackColor" value="FFFFFF" type="2"/>
		<var name="umSlideColor" value="CCCCCC" type="2"/>
		<var name="umShadeAlpa" value="60" type="2"/>
		<var name="umRollColor" value="E8E8E8" type="2"/>
		<var name="umPressColor" value="D4D4D4" type="2"/>
		<var name="umLoadColor" value="DDDDDD" type="2"/>
		<var name="umInfoBar" value="on" type="2"/>
		<var name="umIconColor" value="000000" type="2"/>
		<var name="umButtonColor" value="FFFFFF" type="2"/>
		<var name="umBorderColor" value="000000" type="2"/>
		<var name="svChapterAlpha" value="70" type="2"/>
		<var name="svChapterMenu" value="off" type="2"/>
		<var name="svFlashVersion" value="6" type="2"/>
		<var name="svDetectFlash" value="on" type="2"/>
		<var name="svBufferTime" value="5" type="2"/>
		<var name="svBuffer" value="on" type="2"/>
		<var name="svBufferText" value="Buffering ... " type="2"/>
		<var name="svSetupText" value="Initializing Video" type="2"/>
		<var name="svTitle" value="Golden Pratt PT243" type="1"/>
	</variables>
	<overlay
		file=""
		active="0"
		valign="2"
		halign="0"
	/>
	<events>
	</events>
</SWiSHvideo>
