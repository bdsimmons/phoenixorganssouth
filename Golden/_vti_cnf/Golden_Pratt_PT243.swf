vti_encoding:SR|utf8-nl
vti_timelastmodified:TR|21 Feb 2007 16:21:19 -0000
vti_extenderversion:SR|4.0.2.8912
vti_backlinkinfo:VX|golden/golden_pratt_pt243.html
vti_cacheddtm:TX|21 Feb 2007 16:21:19 -0000
vti_filesize:IR|19334973
