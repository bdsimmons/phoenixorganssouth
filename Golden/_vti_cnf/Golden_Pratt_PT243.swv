vti_encoding:SR|utf8-nl
vti_timelastmodified:TR|21 Feb 2007 16:21:50 -0000
vti_extenderversion:SR|4.0.2.8912
vti_cacheddtm:TX|21 Feb 2007 16:21:50 -0000
vti_filesize:IR|1922
