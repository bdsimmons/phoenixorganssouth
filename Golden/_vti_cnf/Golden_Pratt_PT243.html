vti_encoding:SR|utf8-nl
vti_timelastmodified:TR|21 Feb 2007 16:21:16 -0000
vti_extenderversion:SR|4.0.2.8912
vti_filesize:IR|480
vti_backlinkinfo:VX|
vti_cacheddtm:TX|21 Feb 2007 16:21:16 -0000
vti_cachedlinkinfo:VX|D|clsid:D27CDB6E-AE6D-11cf-96B8-444553540000 S|Golden_Pratt_PT243.swf
vti_cachedsvcrellinks:VX|NDUS|clsid:D27CDB6E-AE6D-11cf-96B8-444553540000 FSUS|Golden/Golden_Pratt_PT243.swf
vti_cachedbodystyle:SR|<body bgcolor=#000000>
vti_cachedhasbots:BR|false
vti_cachedhastheme:BR|false
vti_cachedhasborder:BR|false
